TPP.Core/Setups.cs

```diff
@@ -159,5 +162,43 @@ public static (WebsocketBroadcastServer, OverlayConnection) SetUpOverlayServer(I
                 loggerFactory.CreateLogger<OverlayConnection>(), broadcastServer);
             return (broadcastServer, overlayConnection);
         }
+
+        private static void SetUpDynamicCommands(
+            ILogger logger, CommandProcessor commandProcessor, IResponseCommandRepo responseCommandRepo)
+        {
+            IImmutableList<ResponseCommand> commands = responseCommandRepo.GetCommands().Result;
+
+            HashSet<string> dynamicallyInstalledCommands = new();
+            void InstallCommand(ResponseCommand command)
+            {
+                Command? existing = commandProcessor.FindCommand(command.Command);
+                if (existing != null)
```

Comment:
this seems like a really weird way to do it, because the conflicting commands will be left in the repo and never cleaned up unless done manually, imo that should be done automatically

Reply:
I figured if some static command gets shadowed by a built-in command that it's appropriate that someone would need to take a look at it anyway